content/sensu-go/5.16/guides/monitor-sensu.md

```diff
@@ -0,0 +1,149 @@
+---
+title: "Monitor Sensu with Sensu"
+linkTitle: "Monitor Sensu with Sensu"
+description: "Make sure your Sensu components are properly monitored. This guide describes best practices and strategies for monitoring Sensu."
+weight: 230
+version: "5.16"
+product: "Sensu Go"
+platformContent: False
+menu: 
+  sensu-go-5.16:
+    parent: guides
+---
+
+- [Monitor your Sensu backend instances](#monitor-your-sensu-backend-instances)
+- [Monitor your Sensu agent instances](#monitor-your-sensu-agent-instances)
+- [Monitor your Sensu dashboard instances](#monitor-your-sensu-dashboard-instances)
+- [External etcd PLACEHOLDER](#external-etcd-placeholder)
+- [PostgreSQL PLACEHOLDER](#postgresql-placeholder)
+
+This guide describes best practices and strategies for monitoring Sensu with Sensu.
+It explains how to ensure your Sensu components are properly monitored, including your Sensu backend, agent, API, and dashboard.
+
+To completely monitor Sensu (a Sensu backend with internal etcd and an agent), you will need at least one other independent Sensu instance.
+The second Sensu instance will ensure that you are notified when the primary is down and vise versa.
+
+This guide requires Sensu plugins.
+For more information about using Sensu plugins, see [Install plugins with assets][10].
+
+_**NOTE**: This guide assumes you are not using Sensu clustering, although the strategies in this guide are still useful for monitoring cluster members._
+
+## Monitor your Sensu backend instances
+
+Monitor the host running the `sensu-backend` in two ways:
+
+* Locally by a `sensu-agent` process for operating system checks and metrics and Sensu services that are not part of the Sensu event system.
```

Comment:
Hm, I guess in my mind I had thought about monitoring the API and Dashboard locally. But if we are doing those off machine, I think that's a bit of overkill. All it would let you know about is a firewall issue and you'd have to check for both checks since we don't have aggregates spanning Sensu instances. I'll remove that line.